CareKit/SymptomTracker/OCKSymptomTrackerViewController.m

```diff
@@ -84,8 +84,7 @@ - (void)viewDidLoad {
                                                                               style:UIBarButtonItemStylePlain
                                                                              target:self
                                                                              action:@selector(showToday:)];
-    self.navigationItem.rightBarButtonItem.tintColor = self.progressRingTintColor;
-    
```

Comment:
One thing I noticed is that you are changing the view's tint color, not the progressRingTintColor. 

The progressRingTintColor sets the tintColor for the Ring, the background of the Day Label, and the Today button.

After setting that, if the dev wishes to change the Today's tintColor, they can do so by setting the tintColor on the navigationItem. Although, the tintColor for the progressRing and the dayLabel will always be the same.